.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_ENDPOINT=
```

Comment:
Should this be in a .gitignore, I know we want to have an .env.example

Reply:
^